addons/binding/org.openhab.binding.logreader/README.md

```diff
@@ -0,0 +1,107 @@
+# LogReader Binding
+
+Binding to read and analyze log files. Search patterns are fully configurable, therefore different kind of log files should be possible to monitor by this binding. When certain log events is recognized, openHAB rules can be used to send notification about the event e.g by email for further analyze.
```

Comment:
Do full sentences: "This binding reads and analyses log files".

Reply:
Done